tools/run_tests/artifacts/build_artifact_php.sh

```diff
@@ -19,6 +19,11 @@ cd "$(dirname "$0")/../../.."
 
 mkdir -p "${ARTIFACTS_OUT}"
 
+# build the PHP extension archive (this just zips all the files up)
 pear package
+# test installing the PHP extension via pecl install
```

Comment:
please add a note that files built here are actually not being used in any way (e.g. they are not used in the final package that's being distributed) and the command is here just to give an early signal that php installation may be broken.

Reply:
Done. Updated the comments. Thanks for the review. Will let tests run one more time.